app/screens/settings-screen/default-wallet.tsx

```diff
@@ -5,23 +5,15 @@ import {
 } from "@app/graphql/generated"
 import { useIsAuthed } from "@app/graphql/is-authed-context"
 import { useI18nContext } from "@app/i18n/i18n-react"
-import { ListItem, Text, makeStyles, useTheme } from "@rneui/themed"
+import { Text, makeStyles, useTheme } from "@rneui/themed"
 import * as React from "react"
-import { ActivityIndicator, View } from "react-native"
-import Icon from "react-native-vector-icons/Ionicons"
+import { View } from "react-native"
 import { Screen } from "../../components/screen"
 import { testProps } from "../../utils/testProps"
 import { GaloyIcon } from "@app/components/atomic/galoy-icon"
+import { Select, SelectItem } from "@app/components/select"
 
-const useStyles = makeStyles(({ colors }) => ({
-  viewSelectedIcon: { width: 18 },
-
-  container: { backgroundColor: colors.white },
-
-  text: {
-    color: colors.black,
-  },
-
+const useStyles = makeStyles(() => ({
```

Comment:
we are now putting the style at the bottom (and whenever we touch the style we should move them)